extensions/ql-vscode/src/run-queries-shared.ts

```diff
@@ -495,8 +495,7 @@ export async function createInitialQueryInfo(
   outputDir: QueryOutputDir,
 ): Promise<InitialQueryInfo> {
   const isQuickEval = selectedQuery.quickEval !== undefined;
-  const isQuickEvalCount =
-    selectedQuery.quickEval?.quickEvalCount !== undefined;
+  const isQuickEvalCount = Boolean(selectedQuery.quickEval?.quickEvalCount);
```

Comment:
Alternatively we could simplify further and just pass through `selectedQuery.quickEval?.quickEvalCount` without any manipulation.

The `isQuickEvalCount` field of `InitialQueryInfo` already allows undefined and has a comment saying "Missing is false for backwards compatibility", and therefore all the code consuming this value already has to handle it being `undefined`.

i.e.
```suggestion
  const isQuickEvalCount = selectedQuery.quickEval?.quickEvalCount;
```
or remove this variable entirely and later on do
```typescript
isQuickEvalCount: selectedQuery.quickEval?.quickEvalCount,
```

Reply:
ah yes, that seems like a fine solution :+1: